README.md

```diff
@@ -22,28 +22,19 @@ The .NET Compiler Platform ("Roslyn") provides open-source C# and Visual Basic c
 with rich code analysis APIs.  It enables building code analysis tools with the same APIs 
 that are used by Visual Studio.
 
-### Try new language and IDE features
+### Download C# and Visual Basic
 
-Just want to provide feedback on new [language features](https://github.com/dotnet/roslyn/wiki/Languages-features-in-C%23-6-and-VB-14) 
-and [IDE features](http://blogs.msdn.com/b/visualstudio/archive/2014/11/12/the-c-and-visual-basic-code-focused-ide-experience.aspx)? 
+Want to start developing in C# and Visual Basic? Download [Visual Studio 2015](https://www.visualstudio.com/en-us/downloads/visual-studio-2015-downloads-vs.aspx), 
+which has the latest features built-in. There are also [prebuilt Azure VM images](http://blogs.msdn.com/b/visualstudioalm/archive/2014/06/04/visual-studio-14-ctp-now-available-in-the-virtual-machine-azure-gallery.aspx) available with VS 2015 already installed.
```

Comment:
The link refers to pre-release versions. Try this link instead:
https://azure.microsoft.com/en-us/marketplace/virtual-machines/all/?term=Visual+Studio+2015